src/Pact/Types/Native.hs

```diff
@@ -26,7 +26,7 @@ instance AsString SpecialForm where
   asString Select = "select"
   asString Where = "where"
   asString WithCapability = asString RWithCapability
-  asString Yield = "yield"
+  asString YieldForm = "yield"
```

Comment:
Name clash. RFC what should it be instead?

Reply:
that's fine, or `YieldSF`